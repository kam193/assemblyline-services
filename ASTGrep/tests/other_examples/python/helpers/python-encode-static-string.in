"adgsfdf".encode('ascii')

"adfsgdhf".encode("utf-8")

"adfsgdhf".encode("utf-8", "this is something else")

obj.encode("uth-8")
