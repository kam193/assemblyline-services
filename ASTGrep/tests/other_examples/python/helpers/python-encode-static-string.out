b'adgsfdf'

b'adfsgdhf'

"adfsgdhf".encode("utf-8", "this is something else")

obj.encode("uth-8")
